packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1883,15 +1899,16 @@ class ThemeData with Diagnosticable {
       toggleButtonsTheme: toggleButtonsTheme ?? this.toggleButtonsTheme,
       tooltipTheme: tooltipTheme ?? this.tooltipTheme,
       // DEPRECATED (newest deprecations at the bottom)
-      accentColor: accentColor ?? this.accentColor,
-      accentColorBrightness: accentColorBrightness ?? this.accentColorBrightness,
-      accentTextTheme: accentTextTheme ?? this.accentTextTheme,
-      accentIconTheme: accentIconTheme ?? this.accentIconTheme,
-      buttonColor: buttonColor ?? this.buttonColor,
-      fixTextFieldOutlineLabel: fixTextFieldOutlineLabel ?? this.fixTextFieldOutlineLabel,
-      primaryColorBrightness: primaryColorBrightness ?? this.primaryColorBrightness,
+      accentColor: accentColor ?? _accentColor,
```

Comment:
Added this tweak. `this.<deprecatedProperty>` here does not actually allow folks to stop using it. I just found out in a test case. 😜